tests/system/action/speaker/test_update.py

```diff
@@ -24,10 +24,8 @@ def test_update_correct(self) -> None:
                 },
             }
         )
-        response = self.request("speaker.update", {"id": 890, "marked": True})
+        response = self.request("speaker.update", {"id": 890, "speech_state": "pro"})
         self.assert_status_code(response, 200)
-        model = self.get_model("speaker/890")
```

Comment:
Why did you delete these asserts?

Reply:
Ah, I removed them because originally I just deleted the `marked` field from the payload, so there was nothing to assert. But the validation complains if the request contains no fields, so I already added the `speech_state` field and did not think about adding these back. Will do though